supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -728,6 +740,17 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*: xref:bucket[bucket], xref:connection[connection]
 
+[discrete]
+[[control-plane]]
+==== image:images/yes.png[yes] control plane (noun)
```

Comment:
This isn't alphabetical either

Reply:
Like compute node, control plane also exists in OpenStack-land.